### PR TITLE
Feature/define person classes

### PR DESCRIPTION
**Pull request by williamrolando88**
Hi, in this PR I:
- Created the person, teacher and student classes
- Fix linter errors
- Update readme file

### CODE REVIEW

File: teacher.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+require './person'
+
+class Teacher < Person
+  def initialize(age, specialization, name = 'Unknown')
+    super(age, name)
+    @specialization = specialization
+  end
```
Comment (omar-labana) [inline teacher.rb:7]: - [ ] Kindly override can_use_services? in the `Teacher` class so it always returns true.

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @williamrolando88, 

Excellent work so far! 👌
You have implemented all the required features correctly and without any errors. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: teacher.rb (hunk 1 above)
Comment (williamrolando88) [reply teacher.rb:7]: Done

**Comment by omar-labana** [comment]
# **Approved ✔️**

Hi @williamrolando88,

Congratulations on completing this project! 👏🏼 🎉

You have correctly implemented all of the requirements📃 correctly and without any errors⚠️ or bugs🪲.

Cheers and good luck in your next milestones! 🍀

Feel free to leave any questions or comments in the PR thread(don't forget to tag me 😄) if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**